zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+### Conflicts of interest
+
+If a Constituency has a conflict of interest for a given decision, such as potentially being the recipient of funds or additional authority, then its Voting Units are not eligible to be cast for that vote. This affects the [Approval Threshold] discussed below, which is calculated as a proportion of the remaining Eligible Voting Units.
+
+A Constituency is not automatically considered to have a conflict of interest merely because the decision affects it — for example, when changing its decision-making policy, or because it would need to make changes to software if the Proposal were Approved.
+
+### Approval Thresholds
+
+The Approval Threshold is the absolute majority, as a proportion of the total number of Eligible Voting Units, required for a Proposal to be Approved. As stated earlier, Eligible Voting Units do not include those of Constituencies that have a conflict of interest for this Proposal. However, they do include the Voting Units of Constituencies that have no conflict of interest but choose to abstain.
+
+By default, a Governance Proposal requires a three-fifths Approval Threshold, and MUST have a Deadline at least two weeks from when the Proposal is published on-chain.
+
+All Consituencies MUST be consulted in order to determine whether they consider a draft Proposal to be "controversial", before it published on-chain. If *any* Constituency decides that a Proposal should be treated as controversial, it is then subject to a higher Approval Threshold of three quarters, and a Deadline at least four weeks from when the Proposal is published on-chain. (In the case of the Coin-holder constituency, a coin-holder vote is not necessarily required to establish this; it is likely sufficient to decide based on forum discussions for example.)
+
+The following categories of Governance Proposal are automatically subject to the three-quarters Approval Threshold and 4-week Deadline:
+
+* Increasing or decreasing the voting power of zBloc Constituencies by adjusting how many Voting Units they hold.
+* Increasing or decreasing the number of Funding Pools, the rules that govern them, and the amounts allocated.
+
+In the case of Governance Proposals that affect consensus, final ratification occurs when node operators adopt the software implementing the changes.
+
+Note: Nothing forces developers of Zcash consensus node software to follow this specification. The aim of a process specification like this one is only to establish a social consensus. It fundamentally cannot affect the autonomy of developers of Zcash consensus node software to publish (or not) the software they want to publish, or the autonomy of node operators to run (or not) the software they want to run.
```

Comment:
These facts are also true for the `community-and-coinholder` proposal (or, indeed, any proposal.) They should be copied there.

Reply:
True, but community-and-coinholder is only attempting to handle funding proposals, and so it's obvious in that case that it's not making any demands of node developers.